src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -1719,6 +1737,26 @@ def evaluate(self, data_loader: torch.utils.data.DataLoader, metrics: MetricColl
 
         if self.multi_gpu == MultiGPUMode.DISTRIBUTED_DATA_PARALLEL:
             logging_values = reduce_results_tuple_for_ddp(logging_values, next(self.net.parameters()).device)
+
+        logging_values = get_logging_values(loss_avg_meter, metrics, self.criterion)
```

Comment:
You are overwriting the reduced result for logging_values with the local one when running in DDP (i.e the logging_values in line 1739).

We don't reduce while looping to save time, but in the end of course we'd want the average across processes (metrics are already reduced out of the box since we use torchmetrics, but not the loss loggings).